### ISSUE TITLE
Add. 14,738, foll. 106, 107 | https://bl.vuexistapps.us/manuscript/712 (Part19)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1339]
### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- unwanted character in "some¬what"

### Categories of Physical Description Problems

- [ ] Whitespace
- [X] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### General Notes

This entry has an Additions section, which is empty.

**Comment by mikey-tang16** [comment]
Resolved
extra symbol deleted

Question: Is the empty additions to be deleted? This question also applies to other issues with empty additions. @wlpotter

**Comment by wlpotter** [comment]
Yes, in these cases the entire `additions` element can be deleted

**Comment by mikey-tang16** [comment]
additions deleted
